translations/strings.yaml

```diff
@@ -64,21 +58,155 @@ settingsDev:
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
 
+serversView:
+  searchPlaceholder: Search countries
+  searchNoResultsLabel: No results. Try a different search
+
+# Changing very good to excellent within the in-client feedback form
+feedbackForm:
+  chooseCategory: Choose a category
+  excellentLabel: Excellent
+  # adding new content for the excellent choice in feedback form
+  selectionHeader: Give feedback
+  reviewHeader: Give us 5 stars!
+  reviewBody: If you enjoy using Mozilla VPN, would you review it? Thanks for your support!
+  leaveReviewButton: Leave a Review
+  skipLink: Skip
+
 splittunnel:
   missingAppActionButton:
     value:
```

Comment:
This syntax is valid. If you want, we can have a more compact one:
```
    value: Add apps manually
    comment: Displayed in a notification as “call to action” when apps are missing from the split-tunnel settings.
```

You don't have to change it. I wrote this comment to clarify the sintax of this yaml file :)